### PR TITLE
feat: add build instructions to README

### PR DESCRIPTION
**Pull request by xy2iii**
Add some simple instructions to build the wiki generator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,32 @@
 The goal of this wiki is to modernize the information found in the long-standing OSDev wiki found at [wiki.osdev.org](https://wiki.osdev.org) and to decentralize it. It is currently **WIP** and **not ready** for use.
 
 This wiki is not endorsed by, does not have any connection to, and is not related in any way with another wiki found at [wiki.osdev.org](https://wiki.osdev.org).
+
+# Building
+
+Thanks for contributing!
```
Comment (pitust) [inline README.md:9]: this is a little out of place

File: README.md [hunk 2]
```diff
@@ -3,3 +3,32 @@
...
 
 This wiki is not endorsed by, does not have any connection to, and is not related in any way with another wiki found at [wiki.osdev.org](https://wiki.osdev.org).
+
+# Building
+
+Thanks for contributing! 
+
+## Deps
+
+This wiki is written in Haskell. You will need a [Stack](https://docs.haskellstack.org/en/stable/README/) installation to compile and run the main binary.
```
Comment (pitust) [inline README.md:13]: ... need a Stack installation ... => ... need to have Stack installed ...

File: README.md [hunk 3]
```diff
@@ -3,3 +3,32 @@
...
+
+The project also depends on some non-Haskell deps. To install these, you need a [Python](https://www.python.org/downloads/), [Ruby](https://www.ruby-lang.org/fr/documentation/installation/) and [Rust](https://www.rust-lang.org/tools/install) installation:
```
Comment (pitust) [inline README.md:15]: ... need a Python ... => ... need Python ...

less sure about this one: ... need ... => ... will need ...

**Review by pitust — CHANGES_REQUESTED** [review]

**Comment by ArsenArsen** [comment]
I'd suggest also mentioning `nix`, and providing instructions on how to use the binary cache (`s3://osdev-wiki-cache?scheme=https&endpoint=s3.us-west-000.backblazeb2.com`, but I might swap that out for s3 in the future, if I do so, I'll update the README, no need to worry) as this could significantly cut down on build time. It's also possible that I'll be including adoc, the plugins, pygments and stork in the cache as external deps to make the process further simpler.

Thank you for the writeup!

**Review by xy2iii — COMMENTED** [review]
ok

File: README.md (hunk 1 above)
Comment (xy2iii) [reply README.md]: removed

File: README.md [hunk 4]
```diff
@@ -3,3 +3,68 @@
...
 
 This wiki is not endorsed by, does not have any connection to, and is not related in any way with another wiki found at [wiki.osdev.org](https://wiki.osdev.org).
+
+# Contributing
+
+## Installing dependencies
+
+This wiki is written in Haskell.
+
+The project has some non-Haskell dependencies. To install these, you will need [Python](https://www.python.org/downloads/), [Ruby](https://www.ruby-lang.org/fr/documentation/installation/) and [Rust](https://www.rust-lang.org/tools/install) installed:
```
Comment (ArsenArsen) [inline README.md:13]: One of these links is French.

I also feel like instructing the user on how to get runtimes is unnecessary. All of the other dependencies are distinct projects with their own dependencies, so it should be left up to them to instruct on obtaining them.

File: README.md [hunk 5]
```diff
@@ -3,3 +3,68 @@
...
+- [asciidoctor-bibtex](https://github.com/asciidoctor/asciidoctor-bibtex) - `gem install asciidoctor-bibtex`
+- [stork](https://stork-search.net/docs/install) - `cargo install stork-search --locked`
+
+## Building and running
+
+Because building the Haskell dependecies takes a long time, we recommend using our Nix binary cache to get all binary dependecies without building. (~10 min setup time)
+
+### With Nix
+
+[Install nix](https://nixos.org/download.html#download-nix). Please use the single-user installation as it's easier to set up.
```
Comment (ArsenArsen) [inline README.md:26]: Whether to use single user should be up to the user (same idea as before, dealing with dependencies is out of place for us to instruct, and could only serve to cause confusion).

**Review by ArsenArsen — CHANGES_REQUESTED** [review]
Overall, quite good, thank you for the effort.

Please squash commits into a single `feat(misc): add instructions for the to README` or such when we're done here.

**Review by ArsenArsen — APPROVED** [review]
lgtm, waiting on peer review as I am too sleepy to do proper grammar

**Comment by xy2iii** [comment]
Friendly ping on this

**Review by pitust — APPROVED** [review]
Language is correct (i think...)

**Pull request closed without merging**